templates/documents/general_list.html

```diff
@@ -4,11 +4,14 @@
 {php}$this->assign('GLOBALS', $GLOBALS);{/php}
 
 <link rel="stylesheet" href="{$GLOBALS.css_header}" type="text/css">
+<link rel="stylesheet" href="{$WEBROOT}/public/assets/dropzone-4-3-0/dist/dropzone.css">
```

Comment:
Need to use the public asset path variable (this will allow us to move the public/assets path easily in the future if needed):
``````
<link rel="stylesheet" href="{php}echo $GLOBALS['assets_static_relative']{/php}/dropzone-4-3-0/dist/dropzone.css">